src/pam/README.md

```diff
@@ -3,18 +3,18 @@
 ## Build
 
 ```sh
-meson setup build -Dinih:with_INIReader=true
+meson setup build
```

Comment:
We should explain that INIReader should be installed on system side, right?

Reply:
> We should explain that INIReader should be installed on system side, right?

Yes, I add a instruction for install INIReader now.